package.json

```diff
@@ -7,54 +7,54 @@
+    "debug": "NODE_OPTIONS=--inspect-brk ts-node lib/renovate.ts",
     "doc-fix": "run-s markdown-lint-fix prettier-fix",
     "doc-fence-check": "node tools/check-fenced-code.mjs",
     "lint-documentation": "jest --coverage false test/documentation.spec.ts",
     "eslint": "eslint . --cache --cache-location .cache/eslint --report-unused-disable-directives",
     "eslint-fix": "eslint --cache --cache-location .cache/eslint --fix . --report-unused-disable-directives",
     "eslint-ci": "eslint . --cache --cache-strategy content --cache-location .cache/eslint --format gha",
-    "generate": "run-s generate:*",
+    "generate": "run-s 'generate:*'",
     "generate:imports": "node tools/generate-imports.mjs",
     "git-check": "node tools/check-git-version.mjs",
-    "jest": "cross-env LOG_LEVEL=fatal GIT_ALLOW_PROTOCOL=file jest --logHeapUsage",
-    "jest:14": "run-s \"jest {@}\" --",
-    "jest:16": "run-s \"jest {@}\" --",
-    "jest-debug": "cross-env NODE_OPTIONS=--inspect-brk yarn jest --testTimeout=100000000",
+    "jest": "LOG_LEVEL=fatal GIT_ALLOW_PROTOCOL=file jest --logHeapUsage",
+    "jest:14": "run-s 'jest {@}' --",
+    "jest:16": "run-s 'jest {@}' --",
+    "jest-debug": "NODE_OPTIONS=--inspect-brk jest --testTimeout=100000000",
     "lint": "run-s ls-lint type-check eslint prettier markdown-lint git-check doc-fence-check",
     "lint-fix": "run-s eslint-fix prettier-fix markdown-lint-fix",
     "ls-lint": "ls-lint",
     "markdown-lint": "markdownlint-cli2",
     "markdown-lint-fix": "markdownlint-cli2-fix",
-    "prepare": "run-s prepare:*",
+    "prepare": "run-s 'prepare:*'",
     "prepare:husky": "husky install",
-    "prepare:generate": "run-s generate:*",
+    "prepare:generate": "run-s 'generate:*'",
     "prepare:re2": "npm explore re2 -- npm run install && node tools/check-re2.mjs",
-    "prestart": "run-s generate:* ",
-    "pretest": "run-s generate:* ",
-    "prettier": "prettier --cache --check \"**/*.{ts,js,mjs,json,md,yml}\"",
-    "prettier-fix": "prettier --write --cache \"**/*.{ts,js,mjs,json,md,yml}\"",
+    "prestart": "run-s 'generate:*'",
+    "pretest": "run-s 'generate:*'",
+    "prettier": "prettier --cache --check '**/*.{ts,js,mjs,json,md,yml}'",
+    "prettier-fix": "prettier --write --cache '**/*.{ts,js,mjs,json,md,yml}'",
     "release:prepare": "ts-node tools/generate-docs.ts",
     "release:publish": "node tools/release.mjs",
     "start": "ts-node lib/renovate.ts",
     "test": "run-s lint test-schema jest",
     "test-dirty": "git diff --exit-code",
-    "test-e2e": "run-s test-e2e:*",
-    "test-e2e:pack": "yarn pack",
-    "test-e2e:install": "yarn --cwd test/e2e install --no-lockfile --ignore-optional --prod",
-    "test-e2e:run": "yarn --cwd test/e2e test",
+    "test-e2e": "run-s 'test-e2e:*'",
+    "test-e2e:pack": "npm pack",
+    "test-e2e:install": "cd test/e2e && npm install --no-package-lock --prod",
+    "test-e2e:run": "cd test/e2e && npm test",
```

Comment:
using npm for e2e now, as we also use it for official renovate images.
pnpm doesn't work here yet, probably some interference with the renovate repo.
we can investigate later.